source/utils/adios_reorganize/main.cpp

```diff
@@ -20,7 +20,8 @@
 int main(int argc, char *argv[])
 {
 #if ADIOS2_USE_MPI
-    MPI_Init(&argc, &argv);
+    int provided;
```

Comment:
@chuckatkins I am making this change in every utility/test main functions, this is needed for the `mpi_dp`. I wonder if I should properly wrap this and add it as a function in any of the general ADIOS2 headers?